frontend/src/containers/education/Education.js

```diff
@@ -1,20 +1,18 @@
-import React, { Component } from "react";
+import React, { Component } from 'react'
```

Comment:
As a good practice try to keep the dependencies on the top and separated by a space the app dependencies.